codice/alliance
# CAL-401 updated docs module to inherit jbake.properties file

#### What does this PR do?

replaces local `jbake.properties` file with identical one inherited from DDF

#### Who is reviewing it? 

@mackncheesiest @emmberk @mcalcote @vinamartin @ahoffer 

#### Choose 2 committers to review/merge the PR.
@clockard
@coyotesqrl
@lessarderic
@ricklarsen - Documentation
@rzwiefel
@shaundmorris

#### How should this be tested?

- verify builds successfully (changes are dependent on changes made in this PR: https://github.com/codice/ddf/pull/2841)

#### Any background context you want to provide?

This change will automatically capture any future updates to the jbake template configuration.

#### What are the relevant tickets?

[CAL-401](https://codice.atlassian.net/browse/CAL-401)
[DDF-3532-xxx](https://codice.atlassian.net/browse/CAL-3532)

#### Screenshots (if appropriate)

[documentation.pdf](https://github.com/codice/alliance/files/1632961/documentation.pdf)

#### Checklist:
